recipes-debian/hping3/hping3_debian.bb

```diff
@@ -0,0 +1,31 @@
+inherit debian-package
+require recipes-debian/sources/hping3.inc
+DEBIAN_UNPACK_DIR = "${WORKDIR}/${PN}-${REPACK_PV}"
+
+LICENSE="GPLv2"
```

Comment:
hping3 seems to have GPLv2, BSD-3-Clause and BSD-4-Clause part.
The problem is that GPLv2 and BSD-4-Clause are incompatible.
What's your opinion on this?

Reply:
The license itself is problematic, but it is a problem of hping3. For example, util-linux has same problem. See, https://github.com/kraj/poky/blob/bb1e701fb150647227007c1cd4100cc91de311c1/meta/recipes-core/util-linux/util-linux.inc#L9.
So, I do not think it is dangerous to package it.